packages/vike-react-chakra/README.md

```diff
@@ -0,0 +1,97 @@
+<!-- WARNING: keep links absolute in this file so they work on NPM too -->
```

Comment:
To increase maintainability, let's further minimize the Readme. Let's remove the Vike logo.